PolicyServicePkg/Test/UnitTest/PolicyTest/PolicyTestPei.inf

```diff
@@ -38,4 +38,4 @@
   gPeiPolicyPpiGuid                 ## CONSUMES
 
 [Depex]
-  gPeiPolicyPpiGuid
+  gPeiPolicyPpiGuid AND gEfiPeiMemoryDiscoveredPpiGuid

```

Comment:
This seems reasonable, although different from the description of the PR. I might just note in the description you updated the PEI test depex (does the actual PEI driver need updating, too?).

Reply:
No, just the test as is relies on memory allocation rather then just using the HOB like the real module does.